models/user.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require('mongoose');
+
+const userSchema = new mongoose.Schema(
+    {
+      name: String,
+      email: String,
+      googleId: String,
+    },
+    {
+      timestamps: true,
+    }
+  );
+
+
+module.exports = mongoose.model('User', userSchema);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7812747)
```suggestion
module.exports = mongoose.model("User", userSchema);
```